lib/api/src/js/externals/memory.rs

```diff
@@ -253,46 +231,7 @@ impl Memory {
         Ok(Pages(new_pages))
     }
 
-    /// Return a "view" of the currently accessible memory. By
-    /// default, the view is unsynchronized, using regular memory
-    /// accesses. You can force a memory view to use atomic accesses
-    /// by calling the [`MemoryView::atomically`] method.
-    ///
-    /// # Notes:
-    ///
-    /// This method is safe (as in, it won't cause the host to crash or have UB),
-    /// but it doesn't obey rust's rules involving data races, especially concurrent ones.
-    /// Therefore, if this memory is shared between multiple threads, a single memory
-    /// location can be mutated concurrently without synchronization.
-    ///
-    /// # Usage:
-    ///
-    /// ```
-    /// # use wasmer::{Memory, MemoryView};
-    /// # use std::{cell::Cell, sync::atomic::Ordering};
-    /// # fn view_memory(memory: Memory) {
-    /// // Without synchronization.
-    /// let view: MemoryView<u8> = memory.view();
-    /// for byte in view[0x1000 .. 0x1010].iter().map(Cell::get) {
-    ///     println!("byte: {}", byte);
-    /// }
-    ///
-    /// // With synchronization.
-    /// let atomic_view = view.atomically();
-    /// for byte in atomic_view[0x1000 .. 0x1010].iter().map(|atom| atom.load(Ordering::SeqCst)) {
-    ///     println!("byte: {}", byte);
-    /// }
-    /// # }
-    /// ```
-    pub fn view<T: ValueType>(&self) -> MemoryView<T> {
-        unimplemented!("The view function is not yet implemented in Wasmer Javascript");
-    }
-
-    /// A theoretical alais to `Self::view::<u8>` but it returns a `js::Uint8Array` in this case.
-    ///
-    /// This code is going to be refactored. Use it as your own risks.
-    #[doc(hidden)]
-    pub fn uint8view(&self) -> js_sys::Uint8Array {
+    pub(crate) fn uint8view(&self) -> js_sys::Uint8Array {
```

Comment:
Can we cache the result in an option in the memory?

Doing this operation might be heavy if we do a lot of reads `js_sys::Uint8Array::new(&self.vm_memory.memory.buffer())`